core/src/main/scala/org/http4s/package.scala

```diff
@@ -75,9 +75,10 @@ package object http4s {
       * services will have the opportunity to handle the request.
       * See [[Fallthrough]] for more details.
       */
-    val notFound: Task[Response] = Task.now(Response(Status.NotFound)
-                                               .withAttribute(Fallthrough.fallthroughKey, ())
-                                               .withBody("404 Not Found.").run)
+    val notFound: Task[Response] =
+      Response(Status.NotFound)
+        .withAttribute(Fallthrough.fallthroughKey, ())
+        .withBody("404 Not Found.")
```

Comment:
It's not as fun clean, but I kind of liked memorizing the result of this computation considering its constant and all.